{"identifier":{"url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Authentication\/OAuthProviderError\/Equatable-Implementations","interfaceLanguage":"swift"},"sections":[],"metadata":{"title":"Equatable Implementations","role":"collectionGroup","modules":[{"name":"Authentication"}],"roleHeading":"API Collection"},"kind":"article","hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Authentication","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Authentication\/OAuthProviderError"]]},"schemaVersion":{"minor":3,"major":0,"patch":0},"topicSections":[{"anchor":"Operators","title":"Operators","identifiers":["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Authentication\/OAuthProviderError\/!=(_:_:)"],"generated":true}],"references":{"doc://com.ibm.verify.ios.sdk/documentation/Authentication":{"abstract":[],"title":"Authentication","type":"topic","url":"\/documentation\/authentication","kind":"symbol","role":"collection","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Authentication"},"doc://com.ibm.verify.ios.sdk/documentation/Authentication/OAuthProviderError":{"role":"symbol","url":"\/documentation\/authentication\/oauthprovidererror","title":"OAuthProviderError","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"OAuthProviderError"}],"abstract":[{"text":"A type that indicates when OAuth operartion encounters an error.","type":"text"}],"navigatorTitle":[{"text":"OAuthProviderError","kind":"identifier"}],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Authentication\/OAuthProviderError","kind":"symbol","type":"topic"},"doc://com.ibm.verify.ios.sdk/documentation/Authentication/OAuthProviderError/!=(_:_:)":{"url":"\/documentation\/authentication\/oauthprovidererror\/!=(_:_:)","title":"!=(_:_:)","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"text":"!=","kind":"identifier"},{"text":" ","kind":"text"},{"text":"(","kind":"text"},{"text":"Self","kind":"typeIdentifier"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"Self"},{"text":") -> ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:Sb","text":"Bool"}],"kind":"symbol","type":"topic","abstract":[],"role":"symbol","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/Authentication\/OAuthProviderError\/!=(_:_:)"}}}